.changeset/wise-nails-lick.md

```diff
@@ -2,4 +2,4 @@
 "@nomiclabs/hardhat-ethers": patch
 ---
 
-Adds a custom formatter to better display BigNumber's in Hardhat console (issue #2109).
+Adds a custom formatter to better display BigNumber's in Hardhat console (issue #2109)
```

Comment:
This doesn't belong to this PR, does it?

(If this just means removing a period from the changeset for consistency or something, I don't mind. Just pointing it out in case it means something strange happened during the rebase.)

Reply:
You are correct, this shouldn't be here. I added in this changeset, wondered whether the convention was adding a fullstop, checked the other changesets and found one with a comma and one without and make both of mine consistent with the third.

Reply:
The changeset has already been released as it turns out.